### DISCUSSION TITLE
请问列表删除时使用Popconfirm时怎么使用loading功能？

### DISCUSSION THREAD

**Discussion by xtls2** [ant-design/ant-design#28791] [Q&A]
`        <Popconfirm
          key={"delete"}
          title="确定删除吗？"
          okText="确定"
          cancelText="取消"
        >
          <a href="#">删除</a>
        </Popconfirm>`

我看了文档中的例子，它需要一个变量去保存状态，另一个保存是否显示！
但是列表时可能有很多个Popconfirm 总不能每个Popconfirm都写两个变量吧？

求解。。谢谢

**Comment by John-Yue** [comment]
列表中每一项你不会封装成一个组件？在组件里面引用Popconfirm组件不就好了？
